src/main/java/com/minelittlepony/hdskins/gui/RenderPonyModel.java

```diff
@@ -18,53 +20,67 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.ResourceLocation;
 
+/**
+ * Renderer used for the dummy pony model when selecting a skin.
+ */
 public class RenderPonyModel extends RenderPlayerModel<EntityPonyModel> {
 
-    public RenderPonyModel(RenderManager renderer) {
-        super(renderer);
+    boolean renderingAsHuman = false;
```

Comment:
Ideally, this renderer will never render a human. But since this is for hdskins, changes would need to be made on that side to support that.

Reply:
It will render as a human if you provide an invalid skin, or if the server side has no skin.